src/Apps/Order/Routes/Shipping2/Components/ShippingQuotes2.tsx

```diff
@@ -126,3 +128,42 @@ export const shippingQuoteDisplayNames = {
   standard: "Standard",
   priority: "Priority",
 }
+
+const useAutoSelectBestShippingQuote = (
+  quotes: Array<{
+    readonly id: string
+    readonly isSelected: boolean
+    readonly price: string | null | undefined
+    readonly priceCents: number
+    readonly typeName: string
```

Comment:
pull this from the associated relay type (always) -- never retype things that are auto-generated.